R/generate_qc_report.R

```diff
@@ -17,10 +18,23 @@
 #'
 generate_qc_report <- function(sce,
```

Comment:
```suggestion
generate_qc_report <- function(unfiltered_sce,
```
To avoid confusion do we want to keep the naming convention consistent here with how it is in the actual qc report?

Reply:
We could, though I think my reasoning was that since only one sce is required, I would leave it as it was. Not sure we will ever use it that way, but that was my thought.